2006Redone Server/src/redone/util/Misc.java

```diff
@@ -14,24 +14,9 @@ public static int random(final float range) {
 		return (int) (java.lang.Math.random() * (range + 1));
 	}
 
+	// return a random number from 0 → range - 1
 	public static int randomMinusOne(int range) {
-		int number = (int) (Math.random() * range);
-		return number < 0 ? 0 : number;
-	}
-
-	public static int random(int range1, int range2) { // 0 till range (range
-														// INCLUDED)
-		int keepGoing = 1;
-		int compare1 = 0;
-		while (keepGoing == 1) {
-			compare1 = random(range2);
-			if (compare1 >= range1) {
-				keepGoing = 0;
-			} else {
-				keepGoing = 1;
-			}
-		}
-		return compare1;
+		return (int) Math.random() * range;

```

Comment:
Is this used anywhere else?

Reply:
It does the same thing, just without recursion.

Reply:
True

Reply:
oh wait nvm, this is part of the `randomMinusOne` function,
Math.random returns a value from 0 → 1 (excluding 1), it can return 0 though.
So this should do the exact same.